src/views/components/create-model-form.jsx

```diff
@@ -1,40 +1,102 @@
 import React from 'react';
 import { createModel } from '../../lib/api-client';
-
+import { getAllModels } from '../../lib/api-client'
+import { hashHistory } from 'react-router';
 export default class CreateModelForm extends React.Component {
 
     constructor() {
         super();
         this.state = {
-            id: '',
-            name: ''
+            name: '',
+            description: '',
+            manufacturer: '',
+            vendor: '',
+            location: '',
+            isFaulty: false,
```

Comment:
probably don't want to buy any models/products that have faults...
We shouldn't need to track faults for an entire product.